jekyll/_cci2/use-the-cli-for-config-and-policy-development.adoc

```diff
@@ -87,6 +102,33 @@ circleci policy logs --help
 Get policy (decision) logs
 
 
+Usage:
+  circleci policy logs [flags]
+
+Examples:
+policy logs  --owner-id 462d67f8-b232-4da4-a7de-0c86dd667d3f --after 2022/03/14 --out output.json
+
+Flags:
+      --after string        filter decision logs triggered AFTER this datetime
+      --before string       filter decision logs triggered BEFORE this datetime
+      --branch string       filter decision logs based on branch name
+      --context string      policy context (default "config")
+  -h, --help                help for logs
+      --out string          specify output file name
+      --owner-id string     the id of the policy's owner
+      --project-id string   filter decision logs based on project-id
+      --status string       filter decision logs based on their status
+----
+
+{:.tab.machineblock.Server}
+[source,shell]
+----
+circleci policy logs --help --policy-base-url <your-circleci-server-domain>
+
+# Returns the following:
+Get policy (decision) logs
+
```

Comment:
Nit: Should we put an example for `Server` as well? I am good even if we do not.